asistente_ladm_col/config/expression_functions.py

```diff
@@ -68,6 +68,38 @@ def get_domain_value_from_code(domain_table, code, value_is_ilicode, validate_co
 
     return res
 
+@qgsfunction(args='auto', group='LADM-COL', referenced_columns=[])
+def get_domain_value_from_code_db(db_conn, domain_table, code, value_is_ilicode, validate_conn, feature, parent):
```

Comment:
Please rename db_conn by db_source

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/4888b2f347196a8dc65bad715dc7f8c71a5731dd